packs/spell-effects/spell-effect-enlarge-companion.json

```diff
@@ -0,0 +1,106 @@
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 5
+        },
+        "level": {
+            "value": 2
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Player Core"
+        },
+        "rules": [
+            {
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "PF2E.ActorSizeHuge",
+                        "predicate": [
+                            {
+                                "gte": [
+                                    "parent:level",
+                                    4
+                                ]
+                            }
+                        ],
+                        "value": {
+                            "damage": 4,
+                            "reach": 15,
+                            "size": "huge"
+                        }
+                    },
+                    {
+                        "label": "PF2E.ActorSizeLarge",
+                        "predicate": [
+                            {
+                                "or": [
+                                    {
+                                        "lte": [
+                                            "parent:level",
+                                            3
+                                        ]
+                                    }
+                                ]
+                            }
+                        ],
+                        "value": {
+                            "damage": 2,
+                            "reach": 10,
+                            "size": "large"
+                        }
+                    }
+                ],
+                "flag": "enlarge",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.CreatureSize"
+            },
+            {
+                "key": "CreatureSize",
+                "reach": {
+                    "override": "{item|flags.pf2e.rulesSelections.enlarge.reach}"
+                },
+                "resizeEquipment": true,
+                "value": "{item|flags.pf2e.rulesSelections.enlarge.size}"
+            },
+            {
+                "key": "GrantItem",
```

Comment:
We generally prefer in memory grants - could probably also be modified in the enlarge spell effect.